pkgs/applications/audio/airwindows/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "airwindows-${version}";
+  version = "2020-05-25";
+
+  src = fetchFromGitHub {
+    owner = "airwindows";
+    repo = "airwindows";
+    rev = "897c0830e603a950e838a3753f89334bdf56bb71";
+    sha256 = "1mh7c176fdl5y5ixkgzj2j2b7qifp3lgnxmbn6b7bvg3sv5wsxg3";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
```

Comment:
I see this duplicated over in the already existing `airwave` distribution, just in a different version.

https://github.com/NixOS/nixpkgs/blob/84cf00f98031e93f389f1eb93c4a7374a33cc0a9/pkgs/applications/audio/airwave/default.nix#L16-L26
And a PR with the SDK part updated to the very version you're requiring here, #82380

I think I'd rather see this as a separate derivation, but there were objections to that a long while ago (#10006). I wonder if these still exist? Just commenting on this situation, not requesting any change in particular.

Reply:
``wget https://www.steinberg.net/sdk_downloads/vstsdk368_08_11_2017_build_121.zip`` gives me a 404.